### PR TITLE
Basic Go Programs - Day 6

### PR DESCRIPTION
**Pull request by mrniranjan**
Loops:

Signed-off-by: Niranjan Mallapadi <niranjan@ashoo.in>

### CODE REVIEW

File: loops/loop17.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+				count++
+			}
+		}
+	}
+	fmt.Println(count)
+}
+
+// check if a number is divisible by 7 
+func divBy7(n int) bool {
+	last_digit := n % 10
```
Comment (gsr-shanks) [inline loops/loop17.go]: Use camelcase instead of snake case

File: loops/loop17.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	remaining_digits := n / 10
```
Comment (gsr-shanks) [inline loops/loop17.go]: Use camelcase instead of snake case

File: loops/loop17.go [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+	value := remaining_digits - (last_digit * 2)
+	if value % 7 == 0 {
+		return true
+	} else {
```
Comment (gsr-shanks) [inline loops/loop17.go]: if block ends with a return statement, so drop this else and outdent its block

File: loops/loop17.go [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+	temp := n
+	digits := make([]int, 0, 1)
+	divisible := make([]int, 0, 1)
+	for temp != 0 {
+		digits = append(digits, temp % 10)
+		temp = temp / 10
+	}
+	for i:=2 ; i <= n; i++ {
+		if i == 2 {
+			if ifExists(digits[0], []int{0,2,4,6,8}) == true {
```
Comment (gsr-shanks) [inline loops/loop17.go]: Don't think an explicit "== true" is needed here.

File: loops/loop17.go [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+		}
+		if i == 7 {
+			if divBy7(n) == true {
+				divisible = append(divisible, 7)
+				fmt.Println("Divisible by 7")
+				count++
+			}
+		}
+	}
+	fmt.Println(count)
```
Comment (gsr-shanks) [inline loops/loop17.go]: Why print count here?

File: loops/loop17.go (hunk 4 above)
Comment (mrniranjan) [reply loops/loop17.go]: Pushed the latest code.

**Review by mrniranjan — COMMENTED** [review]
Update the code based on review comments.

**Comment by gsr-shanks** [comment]
LGTM

**Pull request merged**